bench/list_handlers_for_event.ex

```diff
@@ -0,0 +1,134 @@
+defmodule Mix.Tasks.Bench.ListHandlersForEvent do
+  @moduledoc """
+  Runs a benchmark of `list_handlers_for_event/1` callback of various implementations.
+
+  The benchmark spawns processes executing `Events.Impl.list_events_for_event/1` callback in a loop
+  using `Benchee`. The number of spawned processes can be configured using the `--parallelism`
+  option. You can also specify how many handlers will be attached using the `--handlers-count`
+  option. The event invoked during the benchmark is selected so that all attached handlers are be the
+  result of the `list_handlers_for_event/1` call.
+
+  ## Command line options
+
+  * `--parallelism`, `-p` - how many simultaneous processes will be executing the function, defaults
+    to number of core the benchmark is running on
+  * `--handlers-count`, `-h` - how many handlers will be attached, defaults to 100
+  * `--duration`, `-d` - how long the benchmark will run (in seconds), defaults to 10 seconds
+  * `--save` - path to a file where the results of the benchmark will be saved. The same path may
+    be provided in subsequent runs with the `--load` option to compare the new results with the
+    saved ones
+  * `--load` - path to a file where the results will be loaded from
+  * `--save-tag` - "name" for the benchmark results helping to differentiate multiple runs when
+    using the `--load` option. Does not have any effect unless  `--save` is provided as well
+  """
+
+  use Mix.Task
+
+  @shortdoc "Runs a benchmark of `list_handlers_for_event/1` callback of various implementations"
+
+  @switches [
+    parallelism: :integer,
+    duration: :integer,
+    "handlers-count": :integer,
```

Comment:
You can pass `:handlers_count` and the parser will figure it out.

Reply:
Thanks! I didn't know about it, neat 😉